src/JsonApi/Transformer/DocumentTransformer.php

```diff
@@ -74,8 +74,12 @@ private function transformMetaMembers(AbstractDocumentTransformation $transforma
             $transformation->result["jsonapi"] = $jsonApi->transform();
         }
 
-        $meta = array_merge($transformation->document->getMeta(), $transformation->additionalMeta);
```

Comment:
Please remove the unused `array_merge` usage at the beginning of the file, as it fails the CS job: https://github.com/woohoolabs/yin/actions/runs/3558742766/jobs/5979406267#step:7:18

Reply:
Ah I see .. just push first
Sorry for my mistake

Reply:
No, problem! :)